scratch-gui/rspack.config.js

```
@@ -0,0 +1,395 @@
+            ],
+        },
+        hot: true,
+    },
+    output: {
+        library: "GUI",
+        filename: (pathData) =>
+            pathData.chunk.name === IFRAME_EXTENSION_WORKER_ENTRY_NAME
+                ? `js/${IFRAME_EXTENSION_WORKER_ENTRY_NAME}.js`
+                : process.env.NODE_ENV === "production"
+                  ? `js/${CACHE_EPOCH}/[name].[contenthash].js`
+                  : "js/[name].js",
```

ESLint

[error] Arrow function used ambiguously with a conditional expression.

(no-confusing-arrow)